lib/Base/Tensor.cpp

```diff
@@ -449,6 +463,10 @@ void Tensor::init(InitKind init, float val, PseudoRNG &PRNG) {
       getHandle<int64_t>().initXavier(val, PRNG);
       break;
     }
+    case ElemKind::Int8FusedQTy: {
+      getHandle<int8_t>().initXavier(val, PRNG);
```

Comment:
Do want to do anything special to not fill the last 8 bytes?

Reply:
I added a commit to do this along with a test checking that it works. Not sure it'll ever really be used but better to at least handle it correctly.